integration-libs/cdp/root/model/orderDetail/order.ts

```diff
@@ -6,9 +6,11 @@
 
 import { consignments } from './consignments';
 import { entries } from './entries';
-
```

Comment:
This is not the right way to extend the model, Refer to [type-augmentation](https://sap.github.io/spartacus-docs/type-augmentation/)